### DISCUSSION TITLE
Choose re-encoding quality

### DISCUSSION THREAD

**Discussion by 1777777789** [mifi/lossless-cut#1978] [Ideas]
Would be nice to be able to choose the quality of the re-encoded portion with smartcut.
Really love the app thank you very much!

**Comment by mifi** [comment]
Hi. is the quality too low, that's why? which version of losslesscut are you running?

**Reply by 1777777789** [reply]
Hey, well for the type of videos I need this for, the quality is acceptable, however certainly not the same as when it gets to a keyframe. The quality is noticeably much lower. I'm running 3.59.1.
It would be great to have the quality match the rest of the cut or at least improved
Thank you very much
